### PR TITLE
FBXLoader: better support for TGA textures

### LINKED ISSUE #15718: failed to load embed TGA in FBX
##### Description of the problem
Failed to load embed TGA in FBX, reason is set the path in TGA loader

suggest adding below check in LoadTexture function of FBXLoader.js

> 			if ( extension === 'tga' ) {
> 			    console.log(THREE.Loader.Handlers.handlers);
> 				var loader = THREE.Loader.Handlers.get( '.tga' );
> 				console.log(loader);
> 				if ( loader === null ) {
> 
> 					console.warn( 'FBXLoader: TGALoader not found, creating empty placeholder texture for', fileName );
> 
> 					texture = new THREE.Texture();
> 
> 				} else {
>                     //Check embed Media, if true , set path to undefined
> 				    if (fileName.indexOf('blob:') === 0 || fileName.indexOf('data:') === 0) {
> 				        loader.setPath(undefined);
> 				    }
> 					texture = loader.load( fileName );
> 
> 				}

### PR DESCRIPTION
**Pull request by looeee**
Fixes #15718 and simplifies TGA texture support. 

It's now up to the user to set up the TGA loader in their app:

``` js
THREE.Loader.Handlers.add( /\.tga$/i, new TGALoader() );
```

If the `FBXLoader` can't find the `TGALoader` then it creates a dummy or null texture as appropriate.

### CODE REVIEW

**Comment by Mugen87** [comment]
In this case, you can remove the dependency to `TGALoader` from `modularize.js`.

File: examples/js/loaders/FBXLoader.js [hunk 1]
```diff
@@ -288,27 +288,15 @@ THREE.FBXLoader = ( function () {
 
 				case 'tga':
 
-					if ( typeof THREE.TGALoader !== 'function' ) {
+					if( THREE.Loader.Handlers.get( '.tga' ) === null ) {
```
Comment (Mugen87) [inline examples/js/loaders/FBXLoader.js]: Missing space after `if`.

**Comment by Mugen87** [comment]
I think the change is okay however we might want to refactor the injection of loaders in general, see #16771

**Comment by looeee** [comment]
> I think the change is okay however we might want to refactor the injection of loaders in general

Yeah, agreed. If you prefer to skip this change and fix all loaders at once later that's fine by me.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Comment by Mugen87** [comment]
Added this change to the migration guide, too.

https://github.com/mrdoob/three.js/wiki/Migration-Guide#r106--r107

@looeee Feel free to choose a better wording 😇 .

**Comment by looeee** [comment]
@Mugen87 it's great, thank you 👍

**Pull request merged**